public/app/features/alerting/routes.tsx

```diff
@@ -169,6 +169,19 @@ export function getAlertingRoutes(cfg = config): RouteDescriptor[] {
         () => import(/* webpackChunkName: "AlertGroups" */ 'app/features/alerting/unified/AlertGroups')
       ),
     },
+    {
+      path: '/alerting/history/',
+      roles: evaluateAccess([
```

Comment:
we are going to use a new permission. For now, as the feature is behind a ff and it's in progress, we keep this access control.